CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 * Improved testing tags isolation [#320](https://github.com/vmware/go-vcloud-director/pull/320)
 * Added command `make tagverify` to check tags isolation tests [#320](https://github.com/vmware/go-vcloud-director/pull/320)
+* Added methods `Client.GetAccessControl`, `Client.SetAccessControl`
```

Comment:
Please add PR links to the new change log entries.

Reply:
done